benchmark/main.cpp

```diff
@@ -149,8 +149,8 @@ int main(int argc, char *argv[])
             auto name = spec["scenario"].as<std::string>();
             auto ruleset = spec["ruleset"].as<ddwaf_object>();
 
-            ddwaf_config cfg{{0, 0, 0}, {nullptr, nullptr}, nullptr};
-            ddwaf_handle handle = ddwaf_init(&ruleset, &cfg, nullptr);
+            ddwaf_config config{{nullptr, nullptr}, nullptr};
```

Comment:
might as well replace with just `{}`

Reply:
Yeah I'll change it in the next one.

Reply:
Addressed in https://github.com/DataDog/libddwaf/pull/387/commits/95ba3b1f3430ba12ec3061a583669015d0d98deb